packages/mc-html-template/lib/load-env.js

```diff
@@ -1,16 +1,53 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
Do we need this? 🤔

Reply:
Yeah. Otherwise my editor was bonkers and warns things I don't have it warn me about :)